config/SupportedVehicles.xml

```diff
@@ -96,6 +96,33 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPX_816/spx816.xml">
+            <loadingArea offset="0.018 1.39 1.352" width="2.195" height="4.48" length="7.78"/>
```

Comment:
Looks pretty good, I think we should reduce the height a little though:

![image](https://user-images.githubusercontent.com/15822357/165581559-99dc099c-cf76-458c-8589-914f48f81fbe.png)